assets/src/settings-page/mobile-redirection.js

```diff
@@ -0,0 +1,34 @@
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { RedirectToggle } from '../components/redirect-toggle';
+import { Options } from '../components/options-context-provider';
+
+/**
+ * Mobile redirection section of the settings page.
+ */
+export function MobileRedirection() {
+	const { editedOptions } = useContext( Options );
+
+	const { theme_support: themeSupport } = editedOptions || {};
+
+	if ( ! [ 'reader', 'transitional' ].includes( themeSupport ) ) {
```

Comment:
Since mobile redirection is only irrelevant in `standard` mode, we can just call it out more simply:

```suggestion
	if ( 'standard' === themeSupport ) {
```

Reply:
This is written this way because the mobile redirection section is also hidden if the mode is not selected yet (i.e., the first time on the screen), but I made it more readable in f45910b2f